src/app/workflow/workflow.component.ts

```diff
@@ -144,18 +146,22 @@ export class WorkflowComponent extends Entry {
     if (this.isWdl(workflowRef)) {
       const version: WorkflowVersion = this.selectedVersion;
       if (version) {
-        this.workflowsService.secondaryWdl(workflowRef.id, version.name).subscribe((sourceFiles: Array<SourceFile>) => {
-          if (!sourceFiles || sourceFiles.length === 0) {
-            this.workflowsService.wdl(workflowRef.id, version.name).subscribe((sourceFile) => {
-              this.wdlHasHttpImports = importHttpRegEx.test(sourceFile.content);
-            });
-            if (this.enableLaunchWithFireCloud) {
-              this.fireCloudURL = `${Dockstore.FIRECLOUD_IMPORT_URL}/${workflowRef.full_workflow_path}:${version.name}`;
+        if (version !== this.lastVersion || this.lastWorkflow !== workflowRef) {
+          this.lastVersion = version;
+          this.lastWorkflow = this.lastWorkflow;
```

Comment:
?

Reply:
`this.lastWorkflow = workflowRef` maybe?

Reply:
Oops, how embarrassing -- will push new commit soon.